logstash-forwarder.go

```diff
@@ -101,6 +93,11 @@ func main() {
 	}()
 
 	flag.Parse()
+
+	if options.useSyslog {
```

Comment:
we configure syslog if necessary; just after flag parsing to ensure that assertions or options logging goes to syslog.